crates/astria-conductor/src/executor/mod.rs

```diff
@@ -375,40 +377,36 @@ impl Executor {
     /// - if execution or finalization of a block from celestia fails
     async fn handle_executor_command(&mut self, cmd: Option<ExecutorCommand>) -> eyre::Result<()> {
         let Some(cmd) = cmd else {
-            bail!("cmd channel closed unexpectedly; shutting down")
+            bail!("cmd channel closed unexpectedly");
         };
 
         match cmd {
             ExecutorCommand::FromSequencer {
                 block,
             } => {
-                let height = block.header().height.value();
+                let height = block.header().height;
                 let block_subset =
                     SequencerBlockSubset::from_sequencer_block(*block, self.rollup_id);
 
                 match self.execute_block(block_subset).await {
                     Ok(executed_block) => {
                         if let Err(e) = self.update_soft_commitment(executed_block.clone()).await {
-                            let error: &(dyn std::error::Error) = e.as_ref();
-                            error!(height = height, error, "failed to update soft commitment");
+                            let error: &dyn std::error::Error = e.as_ref();
+                            warn!(%height, error, "failed to update soft commitment");
                         }
                     }
                     Err(e) => {
-                        let error: &(dyn std::error::Error) = e.as_ref();
-                        error!(height = height, error, "failed to execute block");
+                        let error: &dyn std::error::Error = e.as_ref();
+                        warn!(%height, error, "failed to execute block");
```

Comment:
same here as above - if we derive an un-executable block, or the execution node fails or whatever reason, that isn't really recoverable

Reply:
failing hard now